app.js

```diff
@@ -26,10 +17,136 @@ const port = 80;
 
 // Webhook route
 app.post("/webhook", function (request, response) {
+
+  let homeId = request.body.home_id;
+  let deviceId = request.body.device_id;
+  let eventType = request.body.event_type;
+  let subType = request.body.sub_type;
+  let eventDescription = getEventDescription(eventType, subType);
+
+  let auth = {
+    client_id: process.env.NETATMO_CLIENT_ID,
+    client_secret: process.env.NETATMO_CLIENT_SECRET,
+    username: process.env.NETATMO_USERNAME,
+    password: process.env.NETATMO_PASSWORD,
+  };
+
+  let api = new netatmo(auth);
+
+  let getHomeData = function(err, data) {
+
+    let homeName = data.homes[0].name;
+    let deviceName = getDevice(deviceId, data.homes[0].smokedetectors).name;
+
+    console.log(`Home: ${homeName}\nDevice: ${deviceName}\nEvent Type: ${eventType}\nDescription: ${eventDescription}`);
+  };
+
+  // Event Listeners
+  api.on('get-homedata', getHomeData);
+
+  // Get home data
+  api.getHomeData();
+
   response.send(request.body);
 });
 
 // Start server
 app.listen(port, () => {
   console.log(`App listening on port ${port}`);
-});
\ No newline at end of file
+});
+
+/**
+ * Get event description
+ */
+function getEventDescription(eventType, subType) {
```

Comment:
This function can be moved to `services/Netatmo.js` so that all the Netatmo functionality is at the same place.